internal/app/config.go

```diff
@@ -39,6 +40,21 @@ type Auth0Config struct {
 	WebClientID  string
 }
 
+type AuthConfig struct {
+	Domain string `default:"http://localhost:8080"`
+	Key    string
+	Pkix   AuthPkixConfig
+}
+
+type AuthPkixConfig struct {
+	Organization  string `default:"Company, INC."`
+	Country       string `default:"US"`
+	Province      string `default:""`
+	Locality      string `default:"San Francisco"`
+	StreetAddress string `default:"Golden Gate Bridge"`
+	PostalCode    string `default:"94016"`
```

Comment:
DN can accept multi strings

```suggestion
	Organization  []string
	Country       []string
	Province      []string
	Locality      []string
	StreetAddress []string
	PostalCode    []string
```